designer/server/src/main/scala/pl/touk/nussknacker/ui/api/ScenarioActivityApiHttpService.scala

```diff
@@ -535,26 +542,84 @@ class ScenarioActivityApiHttpService(
         scenarioActivityRepository.editComment(
           scenarioId,
           ScenarioActivityId(request.scenarioActivityId),
-          request.commentContent
+          validateComment(_, request.commentContent),
```

Comment:
I'm not convinced with the current solution.

I see the following problems:
1. The validation should be done in the core layer and the repository layer should not be bothered with this kind of validation. After we validate some comment we should (at the core layer) produce an instance that holds the invariant that proves (to the repository layer) the instance is validated. 

2. The `addNewComment` method adds sth like a simple/regular comment. The `editComment` method modifies a simple/regular comment and deployment comment (this one has different validation rules). We don't have such distinction at the type level, but IMO we should have it. 

3. Comment smart constructor should have a validation logic encapsulated. IMO too much service/core logic is encapsulated in the repository.